Modules/@babylonjs/react-native/windows/BabylonReactNative/EngineViewManager.h

```diff
@@ -38,35 +35,9 @@ namespace winrt::BabylonReactNative::implementation {
         winrt::Microsoft::ReactNative::ConstantProviderDelegate ExportedCustomBubblingEventTypeConstants() noexcept;
         winrt::Microsoft::ReactNative::ConstantProviderDelegate ExportedCustomDirectEventTypeConstants() noexcept;
 
-        // IViewManagerWithCommands
-        winrt::Windows::Foundation::Collections::IVectorView<winrt::hstring> Commands() noexcept;
-        void DispatchCommand(
-            winrt::Windows::UI::Xaml::FrameworkElement const &view,
-            winrt::hstring const &commandId,
-            winrt::Microsoft::ReactNative::IJSValueReader const &commandArgsReader) noexcept;
-
     private:
-        void OnSizeChanged(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::SizeChangedEventArgs const& args);
-        void OnPointerPressed(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::Input::PointerRoutedEventArgs const& args);
-        void OnPointerMoved(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::Input::PointerRoutedEventArgs const& args);
-        void OnPointerReleased(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::Input::PointerRoutedEventArgs const& args);
-        void OnRendering();
-
         winrt::Microsoft::ReactNative::IReactContext _reactContext{ nullptr };
-        winrt::Windows::UI::Xaml::Controls::SwapChainPanel _swapChainPanel{ nullptr };
-        void* _swapChainPanelPtr{ nullptr };
-        size_t _swapChainPanelWidth{ 1 };
-        size_t _swapChainPanelHeight{ 1 };
-
-        struct RevokerData
-        {
-            winrt::Windows::UI::Xaml::FrameworkElement::SizeChanged_revoker SizeChangedRevoker{};
-            winrt::Windows::UI::Xaml::FrameworkElement::PointerPressed_revoker PointerPressedRevoker{};
-            winrt::Windows::UI::Xaml::FrameworkElement::PointerMoved_revoker PointerMovedRevoker{};
-            winrt::Windows::UI::Xaml::FrameworkElement::PointerReleased_revoker PointerReleasedRevoker{};
-            winrt::event_token RenderingToken;
-        };
-        RevokerData _revokerData{};
+        winrt::BabylonReactNative::EngineView _engineView{ nullptr };

```

Comment:
The only reason this instance is stored is so that it can be replaced, which destroys the old one and unregisters events, right? If/when in the future we add support for multiple view instances, we'll have to update this, and probably do something more like the `OnLoaded`/`OnUnloaded` approach I mentioned, but I guess we can just do it this way for now and worry about fixing it up as needed in the future.

Reply:
yeah we can refactor this as wanted. right now it seems like BabylonNative initialization is driven by the BabylonModule initialie call. I wasn't sure if we wanted the BabylonNative initialization/deinitialization to be driven by the SwapChainPanel OnLoaded/OnUnloaded.

FWIW, there are some lifecycle bugs floating around for UWP support. We can address these sorts of changes in these follow up fixes.